ports/rp2/mpconfigport.h

```diff
@@ -115,6 +123,7 @@
 #define MICROPY_FATFS_LFN_CODE_PAGE             437 /* 1=SFN/ANSI 437=LFN/U.S.(OEM) */
 #define MICROPY_FATFS_RPATH                     (2)
 #if MICROPY_HW_USB_MSC
+#include "hardware/flash.h"
```

Comment:
I don't really like having to move this #include to here, it is a consequence of moving the `MICROPY_HW_USB_MSC` from a commented line in `mpconfigboard.h` to `mpconfigport.h`, can't conditionally include this in the header block at the top.

It seemed desirable to me for both CDC & MSC interfaces to be defined in the same place, especially as MSC is really a port feature not a board-specific feature. However I will undo this part of the change, or do it another way if you prefer.

Reply:
I think this is fine.  Ideally `mpconfigport.h` would include as few files as possible (for compilation speed and build dependency complexity), but this one is necessary for `FLASH_SECTOR_SIZE`.  Putting the include as close as possible to that `FLASH_SECTOR_SIZE` use makes sense.

So, all good!

Reply:
On second look this is a really small header (82 lines, 4 defines, no new dependent headers), so I'm going to move it to the top and include it unconditionally with the other rp2 `hardware/` headers. Not sure why I didn't do that to start. If you'd rather it comes down here then I'll move it back.

Reply:
That's fine, to include it unconditionally.